### ISSUE TITLE
IP/License issue found during Eclipse Legal review

### ISSUE THREAD

**Issue by maxandersen** [jnr/jnr-enxio#10]
JNR Enxio 0.6 - ASL 2.0 Licensed only  - One file contains the LGPL - PollSelectionKey.java.  

``` 
code/jnr-enxio-0.6/src/main/java/jnr/enxio/channels/PollSelectionKey.java
```

**Comment by headius** [comment]
Fixed by 07e4ad5f42bb4ff354480785f0a14e7f5bb9d17e.

**Comment by headius** [comment]
Also a dup of #8.
